indra/newview/llvoavatarself.h

```diff
@@ -299,6 +299,17 @@ class LLVOAvatarSelf :
  **                                                                            **
  *******************************************************************************/
 
+/********************************************************************************
+ **                                                                            **
+ **                    ANIMATIONS
+ **/
+    void                postMotionUpdate() override;  // from llCharacter
+
+ /**                   ANIMATIONS
+  **                                                                            **
+  *******************************************************************************/
```

Comment:
I don't like excessive use of comment delimiters.  When used they should NOT use block-comments, only single-line comments: e.g. `//----------------------------------------------------`.